go.mod

```diff
@@ -4,7 +4,7 @@ go 1.14
 
 require (
 	github.com/dominodatalab/os-release v0.0.0-20190522011736-bcdb4a3e3c2f
-	github.com/go-logr/logr v0.4.0
+	github.com/go-logr/logr v0.1.0
```

Comment:
It's wired. Why downgrade the version of `go-logr` here.

Reply:
operator-sdk does not support logr v0.4.0

Reply:
Got you.